app/src/main/java/com/klinker/android/twitter/activities/compose/Compose.java

```diff
@@ -1040,95 +1050,152 @@ public void run() {
             }, 200);
         }
 
+        /**
+         * Helper method for posting the status update using TweetLonger
+         *
+         * @param twitter the account to tweet from
+         * @return bool true if successful else false
+         */
+        private boolean tweetUsingTweetLonger(Twitter twitter) {
+            boolean isDone = false;
+            TwitLongerHelper helper = new TwitLongerHelper(text, twitter);
+
+            if (notiId != 0) {
+                helper.setInReplyToStatusId(notiId);
+            }
+            if (addLocation) {
+                //waitForLocation();
+                if (waitForLocation()) {
+                    Location location = mLastLocation;
+                    GeoLocation geolocation = new GeoLocation(location.getLatitude(), location.getLongitude());
+                    helper.setLocation(geolocation);
+                }
+            }
+            if (helper.createPost() != 0) {
+                isDone = true;
+            }
+            return isDone;
+        }
+
+        private List<String> getMultipeTweets(String message) {
+            int totalNoTweets = 0;
+            List<String> multiTweets = new Vector<String>();
+            String[] tokens = message.split(" ");
+            String tempString = "";
+            /* Only check for 132 as we are adding (xx/xx) at the end of long tweets */
+            for (int i = 0; i < tokens.length; i++) {
+                if (tempString.length() + tokens[i].length() + 1 <= 132) {
+                    tempString += tokens[i] + " ";
+                } else {
+                    /* We have our split tweet */
+                    multiTweets.add(tempString);
+                    tempString = tokens[i] + " ";
+                }
+            }
+            /* Last tweet will fall out of loop */
+            multiTweets.add(tempString);
+            return multiTweets;
+        }
+
+        /**
+         * Helper function to tweet updates without attaching images. Set the tweet to
+         * scheduled as a workaround for the rate-limit from twitter. Provide a time if the
+         * tweet is scheduled.
+         *
+         * @param twitter The account used to tweet
+         * @param scheduled True if tweet needs to be scheduled
+         */
+        private void tweetWithoutImages(Twitter twitter, boolean scheduled, long time) throws Exception {
```

Comment:
My idea is to allow for the user to schedule a tweet from the compose box itself. And the schedule can be predefined from settings, like every 30 minutes or so. I am working on a PR for the same. It would be nice to keep the function even though not being used in this PR.